pkg/cmd/experimental/ipfailover/ipfailover.go

```diff
@@ -56,6 +61,7 @@ func NewCmdIPFailoverConfig(f *clientcmd.Factory, parentName, name string, out,
 			ErrOut: errout,
 		},
 		ImageTemplate:    variable.NewDefaultImageTemplate(),
+		ServiceAccount:   "ipfailover",
```

Comment:
I deprecated credentials, I added a default service account to make creating an ipfailover act  the same as creating a router. Do we not want that?